Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -241,19 +242,22 @@ private extension CreateBuffersAroundPointsView {
             }
         }
         
-        /// Add a point with its radius to the bufferPoints array.
-        func addBufferPoint() {
+        /// Adds a point with its radius to the buffer points array.
+        /// - Parameters:
+        ///   - point: The center point to create a buffer.
+        ///   - radius: The radius of the buffer.
+        func addBuffer(point: Point, radius: Double) {
```

Comment:
Should `point` have an argument label? I'm thinking `at` makes sense (`addBuffer(at:radius:)`).

Reply:
The purpose of the method is to add a `(point, radius)` tuple to an array rather than to add a graphical buffer at the point, so maybe `addBufferPoint(point:radius:)`?